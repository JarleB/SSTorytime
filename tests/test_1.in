#

-sectioname

::context ::  #...

+::context2 ::

alpha (beta) gamma (delta) epsilon 

  one two three (x99) "27ljdlksjflj"

    " (another) something 

@myalias A (begets) B // let's end with a remark

	one two three (x99)  " (xx) nothing

  three four (dangle) $myalias.2



