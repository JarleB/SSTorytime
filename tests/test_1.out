
------------------------------------
Parsing new file N4Lconfig.in
------------------------------------


1:[31;1;4mN4L N4Lconfig.in No file found in the name N4Lconfig.in at line 1 [0m
